cli/js/diff.ts

```diff
@@ -0,0 +1,222 @@
+// TODO(bartlomieju): this file is copied from std/
```

Comment:
What is the purpose of this "diff" module?

Reply:
I implemented it just like ts-node does; compile TS to JS, diff changes from previous loop pass, execute newly added lines.